core-services/prow/02_config/openshift/cluster-api/_prowconfig.yaml

```diff
@@ -34,7 +38,9 @@ tide:
     repos:
     - openshift/cluster-api
   - includedBranches:
+    - openshift-4.10
     - openshift-4.9
+    - release-4.10
```

Comment:
This does not seem right. This is the `staff-eng-query`, it should only contain 4.11 branches.

Reply:
your comment from above: 

> This update to the complement query is fine, but the staff-eng-approved one needs to have 4.10 branches

Reply:
Yeah, sorry for that. Either a typo or I got confused.